_includes/analytics.html

```diff
@@ -1,3 +1,9 @@
+<!-- Global site tag (gtag.js) - Google Analytics -->
+<script async src="https://www.googletagmanager.com/gtag/js?id=UA-137788077-5"></script>
 <script>
+    window.dataLayer = window.dataLayer || [];
+    function gtag(){dataLayer.push(arguments);}
+    gtag('js', new Date());
 
+    gtag('config', 'UA-137788077-5');
```

Comment:
what Google account did you use for this? Yours?

Reply:
I made and organization temporarily linked to my account, but it can be associated with multiple accounts at the same time.